Jenkinsfile

```diff
@@ -46,31 +47,20 @@ pipeline {
                                 exclusionPattern : '**/*Test.class'
                             ])
                             sh "rm -rf build"
+                            sh "docker-compose -f docker/docker-compose.yml -f docker/docker-compose-tests.yml down"
                         }
                     }
                 }
+
                 stage('Build Docker Images') {
                     steps {
                         script {
-                            if (sh(label: 'Running gp2gp docker build', script: 'docker build -t ${DOCKER_IMAGE} .', returnStatus: true) != 0) {error("Failed to build gp2gp Docker image")}
-                        }
-                    }
-                }
-                stage('Integration Tests') {
-                    steps {
-                        script {
-                            sh '''
-                                docker-compose -f docker-compose-integration-tests.yml build
-                                docker-compose -f docker-compose-integration-tests.yml up --exit-code-from integration_tests
-                            '''
-                        }
-                    }
-                    post {
-                        always {
-                            sh "docker-compose -f docker-compose-integration-tests.yml down --rmi=all"
+                            // TODO: How to make this re-use layers built in test stage? Maybe using docker-compose instead of bare docker would help it find them?
```

Comment:
Do we really need this comment?

Reply:
removed comment and created tech debt story